src/extensions/extension_manager/src/extension_manager.cpp

```diff
@@ -807,11 +812,18 @@ ADUC_Result ExtensionManager::InitializeContentDownloader(const char* initialize
     return result;
 }
 
+DownloadProc ExtensionManager::DefaultDownloadProcResolver(void* lib, const char* symbol)
```

Comment:
why pass in symbol?  There should only be one export that returns a DownloadProc, no?

Reply:
removed